docs/fastdds/use_cases/reduce_memory/reduce_memory.rst

```diff
@@ -0,0 +1,93 @@
+.. include:: ../../../03-exports/aliases.include
+.. include:: ../../../03-exports/aliases-api.include
+
+.. _use-case-reduce-memory:
+
+Reduce memory usage
+===================
+
+A great number of modern systems have tight constraints over available memory, making the reduction of memory usage to a
+minimum critical. Reducing memory consumption of a *Fast DDS* application can be achieved through a different number of
+approaches, mainly through architectural restructuring of the application, but also via limiting the resources the
+middleware utilizes, and by avoiding static allocations.
+
+.. |max_samples| replace:: |ResourceLimitsQosPolicy::max_samples-api|
+.. |max_instances| replace:: |ResourceLimitsQosPolicy::max_instances-api|
+.. |max_samples_per_instance| replace:: |ResourceLimitsQosPolicy::max_samples_per_instance-api|
+.. |allocated_samples| replace:: |ResourceLimitsQosPolicy::allocated_samples-api|
+.. |depth| replace:: |HistoryQosPolicy::depth-api|
+
```

Comment:
As @EduPonz suggests, QoS should be briefly explained here.
Check this again: https://github.com/eProsima/Fast-RTPS-docs/pull/177#discussion_r459352698